pythainlp/tokenize/attacut.py

```diff
@@ -7,16 +7,29 @@
 """
 from typing import List
 
-from attacut import tokenize
+from attacut import Tokenizer
 
+_MODEL_NAME = "attacut-sc"
+_tokenizer = Tokenizer(model=_MODEL_NAME)
 
-def segment(text: str) -> List[str]:
+
+def segment(text: str, model: str = "attacut-sc") -> List[str]:
     """
     Wrapper for AttaCut - Fast and Reasonably Accurate Word Tokenizer for Thai
     :param str text: text to be tokenized to words
+    :param str model:  word tokenizer model to be tokenized to words
     :return: list of words, tokenized from the text
+    :rtype: list[str]
+    **Options for model**
+        * *attacut-sc* (default) using both syllable and character features
+        * *attacut-c* using only character feature
     """
+    global _MODEL_NAME, _tokenizer
     if not text or not isinstance(text, str):
         return []
 
-    return tokenize(text)
+    if model != _MODEL_NAME:
+        _MODEL_NAME = model
```

Comment:
> 
> 
> Changing global variable like this may cause a problem in multi-thread environment.
> 
> It may be possible that thread 1 is setting up "X" model but when it execute the tokenize() function, the model has already been changed to "Y" model by thread 2.
> 
> I'm not a thread expert in Python anyway, but we should better check.

OK. I changed to class.